packages/ubuntu_desktop_installer/lib/pages/installation_complete/installation_complete_page.dart

```diff
@@ -23,63 +23,61 @@ class InstallationCompletePage extends StatelessWidget {
-      builder: (context, lang) => WizardPage(
-        title: Text(lang.installationCompleteTitle),
-        content: Column(
-          children: [
-            Padding(
-              padding: const EdgeInsets.only(top: 64, bottom: 32),
-              child: Container(
-                decoration: BoxDecoration(
-                  shape: BoxShape.circle,
-                  border: Border.all(
-                    color: _kAvatarBorder,
-                    width: 8,
-                  ),
+    final lang = AppLocalizations.of(context);
+    return WizardPage(
+      title: Text(lang.installationCompleteTitle),
+      content: Column(
+        children: [
+          Padding(
+            padding: const EdgeInsets.only(top: 64, bottom: 32),
+            child: Container(
+              decoration: BoxDecoration(
+                shape: BoxShape.circle,
+                border: Border.all(
+                  color: _kAvatarBorder,
+                  width: 8,
                 ),
-                child: CircleAvatar(
-                  radius: 80,
-                  backgroundImage: AssetImage(
-                    'assets/version.png',
-                  ),
+              ),
+              child: CircleAvatar(
+                radius: 80,
+                backgroundImage: AssetImage(
+                  'assets/version.png',
                 ),
               ),
             ),
-            MarkdownBody(
-              data: lang.readyToUse(ProductInfoExtractor().getProductInfo()),
-            ),
-            IntrinsicWidth(
-              child: Column(
-                crossAxisAlignment: CrossAxisAlignment.stretch,
-                children: [
-                  Padding(
-                    padding: const EdgeInsets.symmetric(vertical: 24),
-                    child: ElevatedButton(
-                      onPressed: () {
-                        Provider.of<InstallationCompleteModel>(context,
-                                listen: false)
-                            .reboot(immediate: false);
-                      },
-                      child: Text(
-                        lang.restartInto(
-                            ProductInfoExtractor().getProductInfo()),
-                      ),
-                    ),
-                  ),
-                  OutlinedButton(
+          ),
+          MarkdownBody(
+            data: lang.readyToUse(ProductInfoExtractor().getProductInfo()),
+          ),
+          IntrinsicWidth(
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.stretch,
+              children: [
+                Padding(
+                  padding: const EdgeInsets.symmetric(vertical: 24),
+                  child: ElevatedButton(
                     onPressed: () {
                       Provider.of<InstallationCompleteModel>(context,
                               listen: false)
-                          .shutdown(immediate: false);
+                          .reboot(immediate: false);
```

Comment:
Nice catch!
It would have been better in a separate commit, but I won't be the one to pick nits over this.

Reply:
The diff is really bad and shows these actions shifted. I don't admit hiding any other changes here. :)